README.md

```diff
@@ -184,22 +184,38 @@ This file will then be passed to the underlying Ruby interpreter with
 
 ## Harnesses
 
-You can find several test harnesses in this repository:
+You can find several test harnesses in the `harness/` directory:
 
-* harness - the normal default harness, with duration controlled by warmup iterations and time/count limits
-* harness-bips - a harness that measures iterations/second until stable
-* harness-continuous - a harness that adjusts the batch sizes of iterations to run in stable iteration size batches
-* harness-once - a simplified harness that simply runs once
-* harness-perf - a simplified harness that runs for exactly the hinted number of iterations
-* harness-stackprof - a harness to profile the benchmark with stackprof
-* harness-stats - count method calls and loop iterations
-* harness-vernier - a harness to profile the benchmark with vernier
-* harness-warmup - a harness which runs as long as needed to find warmed up (peak) performance
+* `default` - the normal default harness, with duration controlled by warmup iterations and time/count limits
+* `bips` - a harness that measures iterations/second until stable
+* `continuous` - a harness that adjusts the batch sizes of iterations to run in stable iteration size batches
+* `once` - a simplified harness that simply runs once
+* `perf` - a simplified harness that runs for exactly the hinted number of iterations
+* `stackprof` - a harness to profile the benchmark with stackprof
+* `stats` - count method calls and loop iterations
+* `vernier` - a harness to profile the benchmark with vernier
+* `warmup` - a harness which runs as long as needed to find warmed up (peak) performance
+* `chain` - a harness to chain multiple harnesses together
+* `mplr` - a harness for multiple iterations with time limits
 
-To use it, run a benchmark script directly, specifying a harness directory with `-I`:
+To use a specific harness, run a benchmark script directly with `-I` to add the harness directory to the load path, and `-r` to require the specific harness:
 
 ```
+# Use default harness
 ruby -Iharness benchmarks/railsbench/benchmark.rb
+
+# Use the 'once' harness
+ruby -Iharness -ronce benchmarks/railsbench/benchmark.rb
+
+# Use the 'perf' harness
+ruby -Iharness -rperf benchmarks/railsbench/benchmark.rb
```

Comment:
Doesn't this approach cause issues for `stackprof` and `vernier` which are both harness names and gem names?
Specifically I'm concerned the "nested" `require` might just do nothing.

Reply:
I tried on this branch and indeed it seems problematic (tested on `ruby 3.4.7`):
```
$ ruby -Iharness -rstackprof benchmarks/fib.rb
Adding to load path: /home/eregon/code/yjit-bench/harness
/home/eregon/code/yjit-bench/harness/stackprof.rb:72:in `run_benchmark': uninitialized constant StackProf (NameError)

  StackProf.run(out: out, **opts) do
  ^^^^^^^^^
	from benchmarks/fib.rb:11:in `<main>'
```
and
```
$ ruby -Iharness -rvernier benchmarks/fib.rb
Adding to load path: /home/eregon/code/yjit-bench/harness
/home/eregon/code/yjit-bench/harness/vernier.rb:19:in 'Object#run_benchmark': uninitialized constant Vernier (NameError)

  Vernier.profile(out: out) do
  ^^^^^^^
	from benchmarks/fib.rb:11:in '<main>'
```

Reply:
Yes. I changed the approach a little bit. We should not need to change the load path, and instead use relative requires. To help the developer experience, I changed the `run_once` script to parse a `--harness` argument.